packages/server/database/migrations/20170926142800-freemium.js

```diff
@@ -18,13 +17,13 @@ exports.up = async (r) => {
         r
           .table('Organization')
           .get(org('id'))
-          .replace((row) => row.merge({tier: PERSONAL}).without('stripeId', 'stripeSubscriptionId'))
+          .replace((row) => row.merge({tier: 'personal'}).without('stripeId', 'stripeSubscriptionId'))
```

Comment:
Any reason these went from vars to hardcoded strings?

Reply:
got rid of the old constants so had to nuke everything that had a ref to them